pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -148,6 +148,37 @@ func (p *provider) getConfig(s v1alpha1.ProviderSpec) (*Config, *providerconfigt
 		klog.V(6).Infof("Region from configuration or environment variable not found")
 	}
 
+	// Load timeout
+	instanceReadyCheckPeriodStr, err := p.configVarResolver.GetConfigVarStringValue(rawConfig.InstanceReadyCheckPeriod)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to get the value of \"InstanceReadyCheckPeriod\" field, error = %v", err)
+	}
+
+	c.InstanceReadyCheckPeriod, err = time.ParseDuration(instanceReadyCheckPeriodStr)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to parse the value of \"InstanceReadyCheckPeriod\" field, error = %v", err)
+	}
+
+	if c.InstanceReadyCheckPeriod < 60*time.Second {
+		c.InstanceReadyCheckPeriod = 120 * time.Second
+	}
+
+	instanceReadyCheckTimeoutStr, err := p.configVarResolver.GetConfigVarStringValue(rawConfig.InstanceReadyCheckTimeout)
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to get the value of \"InstanceReadyCheckTimeout\" field, error = %v", err)
+	}
+
+	c.InstanceReadyCheckTimeout, err = time.ParseDuration(instanceReadyCheckTimeoutStr)
+
+	c.InstanceReadyCheckPeriod, err = time.ParseDuration(instanceReadyCheckPeriodStr)
```

Comment:
```suggestion
```

is already parsed and error is handled previously.

Reply:
ufff. copy/paste error... Thanks for pointing this one out!